### ISSUE TITLE
I can't wrench in survival and casting icons is white.

### ISSUE THREAD

**Issue by K-Kroon** [TeamOpenIndustry/ImmersiveRailroading#1091]
**Describe the bug**  
1, Icons in Casting is white.

2, I can't wrench a train together. It only sounds lika a click when I try.

**How to reproduce**  
Steps to reproduce the unwanted behavior:

 1. Go to ...
 2. Do ...
 3. ...
 
**Expected behavior**  
Icons in casting is white and I can't wrench a train.

**Screenshots**  
![mc1](https://user-images.githubusercontent.com/91799114/135727245-489426d6-3357-40cf-bf8d-9cf69632acd2.png)
![mc2](https://user-images.githubusercontent.com/91799114/135727377-fb6c891f-4a82-4f7a-a8fb-dd51ad61157f.png)

**System Information**  
OS: Windows 10
System Memory: 16gb
Allocated Memory: 10gb
CPU: Intel I7 4720HQ
GPU: Internal (Laptop)
IR-Version: 1.12.2-1.9.1
Forge-Version: 1.12.2 - 14.23.5.2855

**Modlist**  
![mc3](https://user-images.githubusercontent.com/91799114/135727411-5e8c24bf-ebad-4241-ab29-c372f479b734.png)
 
**latest.log**
Do not have a log file for this error

**Additional context**  
-

**Comment by Larky2k** [comment]
Clear cache folder for icon issue, remove Railstuff for second issue

**Comment by K-Kroon** [comment]
Clearing cache folder fixed almost the first issue. Looks better.

I removed railstuff but there is no diffrence with the second issue. When i uses the wrench it dosen't say anything.

**Comment by Larky2k** [comment]
Wait are you trying to hook two rollingstock together then you nee to use coupling hook

**Comment by K-Kroon** [comment]
No. I have created SW1500 frame, bogey, wheels and so on. I put the frame on the track, and then I have the other items in my inventory and using the wrench. But nothing is mounted. If I take the bogey and trying to add it to the frame, nothing happendes.

**Comment by Larky2k** [comment]
have you got all parts including plates

**Comment by K-Kroon** [comment]
No. I have played with an old version of IR before in a old world. This time I upgraded all mod versions and forge and started a new one. I have got so far I have the casting, plate machine, rails and steam hammer. In the old version, when I used the wrench it says I was missing plate, bogey and so on. But now I don't get any of that information. It says nothing.

**Comment by Larky2k** [comment]
what java you on?

**Comment by K-Kroon** [comment]
In this case it's Java 8.0.3010.9. I also have jdk 16.0.2 installed that i'm using for the server. Should I use jdk instead?

**Comment by Larky2k** [comment]
needs to be java 8
